T/48: CKFinder command improvements

jodator opened the pull request:
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: CKFinderCommand should work with either 'link' or 'imageInsert' commands not available. Closes #48.

---

### Additional information

* First commits fixes the issue with `'imageUpload'` command use instead of `insertImage'`.
* The latter part enables CKFinderCommand to work without one of: link or image insert commands. So if link command is not available and user selects a non-image file then a warning is displayed.

Reinmar commented:
I've got a problem with what to log and to whom. It does not make sense to log to the user that a command isn't enabled. We should log something more digestible.

* Cannot insert an image. This editor does not support inserting images.
* Cannot insert a link to a file. This editor does not support links.

But they are awfully long and it does not sound like a popular case that someone may have them disabled. Maybe we can do it the other way around and make both `Image` and `Link` requirements of CKFinder and see if anyone will complain?

WDYT?

jodator commented:
> Maybe we can do it the other way around and make both `Image` and `Link` requirements of CKFinder and see if anyone will complain?

Sure It seems like the most common scenario. If we see complaints we could try to provide some solution then.

jodator commented:
@Reinmar So the code get clearer by removing those warnings. I've added the `Image` and `Link` plugins as required ones for the command. After that, it also does not makes sense to check if a command is available - so those checks are also removed.
